jobs/haproxy/spec

```diff
@@ -580,6 +581,13 @@ properties:
       cidr_whitelist:
       - 172.168.4.1/32
       - 10.2.0.0/16
+  ha_proxy.expect_proxy:
```

Comment:
The property name is for me too general. I would expect boolean value for such property. Maybe use`expect_proxy_cidrs`. It is more clear.

Reply:
Good suggestion. It was discussed beforehand and the name was selected to ensure the similarity to accept_proxy. I would change it, but need one more opinion about it @peanball.

Reply:
I'm with @peanball on this one. I think we should keep it as is. `accept_proxy` is boolelan, `expect_proxy_cidrs` ist a list of CIDRS

Reply:
Rename done